### ISSUE TITLE
trouble when loading pre-trained model

### ISSUE THREAD

**Issue by serengil** [yule-li/CosFace#22]
Hello,

When I try to restore the model from Google Drive link with named model-20180626-205832.ckpt-60000 it got the following error.

Unable to open table file models\model-20180626-205832.ckpt: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
         [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

tensorflow: 1.9.0
python: 3.5.5

I think that tensorflow requires .data, .index and .meta files in the same directory. However, google drive link just downloads "model-20180626-205832.ckpt-60000.data-00000-of-00001"
